AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/utils.py

```diff
@@ -127,6 +127,32 @@ def wait_for_condition(function, timeout_in_seconds):
                 if ret:
                     return True
 
+    @staticmethod
+    def after_level_load() -> None:
+        """
+        Updates viewport, closes helper gizmos, error reports, and error logs.
+        Closes out FPS meters and disables anti-aliasing.
+        :return: None
+        """
+        # Give everything a second to initialize.
+        general.idle_enable(True)
+        general.idle_wait(1.0)
+        general.update_viewport()
+        general.idle_wait(0.5)  # half a second is more than enough for updating the viewport.
+
+        # Close out problematic windows, FPS meters, and anti-aliasing.
+        if general.is_helpers_shown():  # Turn off the helper gizmos if visible
+            general.toggle_helpers()
+            general.idle_wait(1.0)
+        if general.is_pane_visible("Error Report"):  # Close Error Report windows that block focus.
+            general.close_pane("Error Report")
+        if general.is_pane_visible("Error Log"):  # Close Error Log windows that block focus.
+            general.close_pane("Error Log")
+        general.idle_wait(1.0)
+        general.run_console("r_displayInfo=0")
+        general.run_console("r_antialiasingmode=0")
+        general.idle_wait(1.0)
+
```

Comment:
This is a very specific set of steps for being a generic util function, and the name does not describe what it does.

I suggest splitting it by functionality in multiple functions with descriptive names.

Reply:
true